lib/foreman_remote_execution_core.rb

```diff
@@ -3,18 +3,26 @@
 module ForemanRemoteExecutionCore
   extend ForemanTasksCore::SettingsLoader
   register_settings([:remote_execution_ssh, :smart_proxy_remote_execution_ssh_core],
-                    :ssh_identity_key_file => '~/.ssh/id_rsa_foreman_proxy',
-                    :ssh_user              => 'root',
-                    :remote_working_dir    => '/var/tmp',
-                    :local_working_dir     => '/var/tmp',
-                    :kerberos_auth         => false)
+                    :ssh_identity_key_file   => '~/.ssh/id_rsa_foreman_proxy',
+                    :ssh_user                => 'root',
+                    :remote_working_dir      => '/var/tmp',
+                    :local_working_dir       => '/var/tmp',
+                    :kerberos_auth           => false,
+                    :async_ssh               => false,
+                    :runner_refresh_interval => nil)
```

Comment:
Let's just put a comment here about why we put `nil` here (to set the default based on the async_ssh true/false` later)